plugins/source/gcp/client/multiplex.go

```diff
@@ -13,3 +15,38 @@ func ProjectMultiplex(meta schema.ClientMeta) []schema.ClientMeta {
 	}
 	return l
 }
+
+func OrgMultiplex(meta schema.ClientMeta) []schema.ClientMeta {
+	client := meta.(*Client)
+
+	projectsClient, err := resourcemanager.NewProjectsClient(client.ctx, client.ClientOptions...)
+	if err != nil {
+		client.Logger().Error().Err(err).Msg("OrgMultiplex: Failed to create ProjectsClient")
+		return []schema.ClientMeta{client}
+	}
+
+	dupes := map[string]struct{}{}
+
+	l := make([]schema.ClientMeta, 0, len(client.projects))
```

Comment:
Do we want to provide some type of logging (even if it is just `debug`) so that users can be aware that CQ is not hanging?

Reply:
@bbernays erm, we have "skipping project" and "found project" messages on lines 43 and 47 already?